### ISSUE TITLE
[FEAT]: Expert Train & Inference for Sarc dataset with Mistral

### ISSUE THREAD

**Issue by Jasonqi146** [lwaekfjlk/mmodel-soup#89]
### Description

Use Sarc training split of R, U, and S to train model Mistral seperately and save to

Sarc R finetuned Mistral
Sarc U finetuned Mistral
Sarc S finetuned Mistral
Then record the performance of the three models on the test set.
